README.rst

```diff
@@ -1,6 +1,46 @@
 Demos
 =====
-A collection of coding examples using the D-Wave Ocean SDK.
+A collection of demos using the D-Wave Ocean SDK.
+
+**NOTE** Each demo is stored in its own subdirectory and may have its own
+set of package requirements (for example, `antenna-selection/requirements.txt
+<antenna-selection/requirements.txt>`_).
+
+Setup Instructions
+------------------
+As each demo may have a different set of package requirements, you may want to
+set up a virtual environment to contain said packages and run your demo within
+that environment.
+
+Virtual environment setup and package installation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+::
+
+  git clone git@github.com:dwavesystems/demos.git
+  cd demos/<desired-demo>
+  python3 -m venv <desired-demo>_env
```

Comment:
Since you're already chdir-ing into the `<desired-demo>`, why complicate with naming the env to include `<desired-demo>` prefix? Why not:
```
python3 -m venv env
```